### PR TITLE
DOCS: Install raspbian updates - for 22.1

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**
Porting:
https://github.com/openvinotoolkit/openvino/pull/13798

Ticket:
CVS-95392

### CODE REVIEW

File: docs/install_guides/installing-openvino-raspbian.md [hunk 1]
```diff
@@ -26,38 +26,38 @@
...
+2. Create an installation folder for OpenVINO. If the folder already exists, skip this step.
+   ```sh
+   sudo mkdir -p /opt/intel
+   ```
+   > **NOTE**: The `/opt/intel` path is the recommended folder path for administrators or root users. If you prefer to install OpenVINO in regular userspace, the recommended path is `/home/<USER>/intel`. You may use a different path if desired.
+
+3. Go to your `~/Downloads` directory and download OpenVINO Runtime archive file for Debian from [OpenVINO package repository](https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.1/).
    ```sh
    cd ~/Downloads/
+   sudo wget https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.1/l_openvino_toolkit_runtime_rhel8_p_2022.1.0.643.tgz -O openvino_2022.1.0.643.tgz
```
Comment (ilya-lavrenov) [inline docs/install_guides/installing-openvino-raspbian.md:46]: why rhel? while we are about documentation for raspbian

Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-raspbian.md:46]: Thanks for pointing that out. Correction is under way.

**Review by ilya-lavrenov — APPROVED** [review]

File: docs/install_guides/installing-openvino-raspbian.md [hunk 2]
```diff
@@ -26,38 +26,38 @@
...
 
 1. Open the Terminal or your preferred console application.
-2. Go to the directory in which you downloaded the Intel® Distribution of OpenVINO™ toolkit. This document assumes this is your `~/Downloads` directory. If not, replace `~/Downloads` with the directory where the file is located.
+2. Create an installation folder for OpenVINO. If the folder already exists, skip this step.
+   ```sh
+   sudo mkdir -p /opt/intel
+   ```
+   > **NOTE**: The `/opt/intel` path is the recommended folder path for administrators or root users. If you prefer to install OpenVINO in regular userspace, the recommended path is `/home/<USER>/intel`. You may use a different path if desired.
+
+3. Go to your `~/Downloads` directory and download OpenVINO Runtime archive file for Debian from [OpenVINO package repository](https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.1/).
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-raspbian.md:43]:
```suggestion
3. Go to your `~/Downloads` directory and download OpenVINO Runtime archive file for Debian from [OpenVINO package repository](https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.1.1/).
```

File: docs/install_guides/installing-openvino-raspbian.md [hunk 3]
```diff
@@ -26,38 +26,38 @@
...
    ```sh
-   sudo mkdir -p /opt/intel/openvino_2022
+   sudo tar -xf openvino_2022.1.1.7030.tgz
+   sudo mv l_openvino_toolkit_debian9_arm_2022.1.1.7030.39aba80957e /opt/intel/openvino_2022.1.1.7030
    ```
-4. Unpack the archive:
+5. For simplicity, it is useful to create a symbolic link as below:
    ```sh
-   sudo tar -xf l_openvino_toolkit_runtime_raspbian_p_<version>.tgz --strip 2 -C /opt/intel/openvino_2022
+   sudo ln -s openvino_2022.1.1.7030 openvino_2022
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-raspbian.md:55]:
```suggestion
   sudo ln -s openvino_2022.1.1.7030 openvino_2021
```

**Comment by xu-yuan1** [comment]
Hi @ilya-lavrenov , since 2022.1 does not have the Debian package, should we ask users to directly download 2022.1.1 package for Raspbian installation?

**Pull request merged**